internal/jobqueue/fsjobqueue/fsjobqueue.go

```diff
@@ -253,6 +260,10 @@ func (q *fsJobQueue) FinishJob(id uuid.UUID, result interface{}) error {
 		return fmt.Errorf("error writing job %s: %v", id, err)
 	}
 
+	q.mapMu.Lock()
```

Comment:
I think the same applies to this patch, that we can just use `mu` instead of `mapMu`.